### DISCUSSION TITLE
[@types/jquery-toast-plugin] is missing a simple string overload.

### DISCUSSION THREAD

**Discussion by GalLalouche** [DefinitelyTyped/DefinitelyTyped#70880] [Issues with a @types package] [answered]
According to the [documentation](https://github.com/kamranahmedse/jquery-toast-plugin), `$.toast("hi")` is perfectly valid code, but according to the defined interface, you would have to pass in an object, e.g., `$.toast({text: "hi"})`.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
I just opened #70881 to fix this

**Reply by GalLalouche** [reply]
Thank you!
